src/main/kotlin/org/rust/ide/inspections/import/AutoImportFix.kt

```diff
@@ -122,16 +121,6 @@ class AutoImportFix(element: RsElement, private val context: Context) :
             }
 
             val referenceName = basePath.referenceName ?: return null
-
-            val isNameInScope = path.hasInScope(referenceName, TYPES_N_VALUES) && path.parent !is RsMacroCall
-            if (isNameInScope) {
```

Comment:
This check can be removed because it is checkd in `ImportContext2.hasVisibleItemInRootScope`